extensions/amazon-dynamodb/deployment/src/test/resources/sync-apache-broken-config.properties

```diff
@@ -1,3 +1,4 @@
+quarkus.dynamodb.sync-client.type = APACHE
```

Comment:
So for enums, the recommended way is to use lower case, dashed versions in the `application.properties`. They will be automatically converted. Typically the documentation will contain `apache` and `url` so let's be consistent.

Reply:
will clean it up